oauthproxy.go

```diff
@@ -686,6 +701,12 @@ func (p *OAuthProxy) isAllowedRoute(req *http.Request) bool {
 	return false
 }
 
+// IsForwardedRequest is used to check if X-Forwarded-Host header exists or not
+func (p *OAuthProxy) IsForwardedRequest(req *http.Request) bool {
+	isForwarded := req.Host != util.GetRequestHost(req)
+	return isForwarded
+}
+
```

Comment:
Does this need to be exported? 
```suggestion
// isForwardedRequest is used to check if X-Forwarded-Host header exists or not
func (p *OAuthProxy) isForwardedRequest(req *http.Request) bool {
	isForwarded := req.Host != util.GetRequestHost(req)
	return isForwarded
}

```